activerecord/test/cases/relation/update_all_test.rb

```diff
@@ -241,6 +241,38 @@ def test_touch_all_cares_about_optimistic_locking
     end
   end
 
+  def test_klass_level_update_all
```

Comment:
I think we should use `class` instead of `klass` here.

We generally write `klass` when it would conflict with the Ruby keyword `class` but that will not happen here. 

Also, we use `class` in other similar test names:
https://github.com/rails/rails/blob/808c589fd883d87da720e3ffc103437bdf7e457e/activerecord/test/cases/persistence_test.rb#L56
https://github.com/rails/rails/blob/808c589fd883d87da720e3ffc103437bdf7e457e/activerecord/test/cases/persistence_test.rb#L70
https://github.com/rails/rails/blob/808c589fd883d87da720e3ffc103437bdf7e457e/activerecord/test/cases/persistence_test.rb#L943
https://github.com/rails/rails/blob/808c589fd883d87da720e3ffc103437bdf7e457e/activerecord/test/cases/persistence_test.rb#L954
https://github.com/rails/rails/blob/808c589fd883d87da720e3ffc103437bdf7e457e/activerecord/test/cases/persistence_test.rb#L966
https://github.com/rails/rails/blob/808c589fd883d87da720e3ffc103437bdf7e457e/activerecord/test/cases/persistence_test.rb#L975